### PR TITLE
fix(dipatcher): add config validation for KubernetesManager

### PR DESCRIPTION
**Pull request by kkaneda**

### CODE REVIEW

File: dispatcher/internal/config/config.go [hunk 1]
```diff
@@ -68,6 +68,19 @@ type KubernetesManagerConfig struct {
 	PprofBindAddress   string `yaml:"pprofBindAddress"`
 }
 
+func (c *KubernetesManagerConfig) validate() error {
+	if c.EnableLeaderElection && c.LeaderElectionID == "" {
+		return fmt.Errorf("leader election ID must be set")
+	}
+	if c.MetricsBindAddress == "" {
```
Comment (Ladicle) [inline dispatcher/internal/config/config.go:75]: metrics & health bind address have a default value, so it works if unspecified

**Review by Ladicle — APPROVED** [review]

File: dispatcher/internal/config/config.go (hunk 1 above)
Comment (kkaneda) [reply dispatcher/internal/config/config.go:75]: Ah i see.

Health probe is disabled when unspecified? https://github.com/kubernetes-sigs/controller-runtime/blob/main/pkg/manager/manager.go#L228

Comment (kkaneda) [reply dispatcher/internal/config/config.go]: Anyhow removed the both checks.

Comment (aya-cloudnatix) [reply dispatcher/internal/config/config.go]: Ah, yes. The health probe is disabled when unspecified. Only metrics have a default value.

**Pull request merged**